Hello Kids/Hello.generated.cs

```diff
@@ -1 +1,23 @@
-﻿
\ No newline at end of file
+//------------------------------------
+//
+// This file is autogenerated.
+//
+//------------------------------------
+namespace Hello
+{
+    public enum StoredNames
+    {
+        /*
+            I am going to need to "fill in" the values from the json file here.
+            I will have to use a T4 Expression Control block
+        */
```

Comment:
Is this comment meant to show up in this generated file or just be a comment in the tt file?

Reply:
It is in the T4 and Shows up in the cs file. Shows writing out comments to code.